README.md

```diff
@@ -68,6 +68,23 @@ configured previously on page "New settings - Link checker robot".
 
 Once bot user is created "Bot user" line in status report should be showing "Good".
 
+## Disabling crawling of specific course categories
+
+This is achieved by configuring proper security roles in Moodle and assigning these roles to the robot user on desired categories.
+
+Import role "Robot" from local/linkchecker_robot/robot.xml on Site administration ► Users ► Permissions ► Define roles ► Add a new role.
```

Comment:
these are a little hard to read inline, can you put each crumb trail into a new line